<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uploadFileDialog</class>
 <widget class="QDialog" name="uploadFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>295</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>90</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Upload file</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gui.qrc">
    <normaloff>:/images/upload.png</normaloff>:/images/upload.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>14</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>30</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelPath">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Path:</string>
        </property>
        <property name="buddy">
         <cstring>lineEditPath</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditPath">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonBrowseFile">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>116</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="labelRecordSize">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Choose record size:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton30">
        <property name="text">
         <string>30</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton64">
        <property name="text">
         <string>64</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton98">
        <property name="text">
         <string>98</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton200">
        <property name="text">
         <string>200</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>116</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="labelFitOption">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Choose Fit option:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QRadioButton" name="radioButtonFirstFit">
        <property name="text">
         <string>First Fit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonBestFit">
        <property name="text">
         <string>Best Fit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonWorseFit">
        <property name="text">
         <string>Worse Fit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonOK">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>141</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="labelSetFileName">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Set file name:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditPath</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditFileName"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>141</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="labelSetOwnerName">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Set owner name:</string>
      </property>
      <property name="buddy">
       <cstring>lineEditPath</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditOwnerName"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonOK</sender>
   <signal>clicked()</signal>
   <receiver>uploadFileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>uploadFileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
